meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,160 @@
+    - Reviewed perf PR for `datastore-fs` to replace `write-file-atomic` instead of `fast-write-atomic`
+        - Reviewed `fast-write-atomic` code, opened 2 issues [#3](https://github.com/mcollina/fast-write-atomic/issues/3) and [#4](https://github.com/mcollina/fast-write-atomic/issues/3) which were resolved quickly by @mcollina
+    - Helped with reviews and releases necessary for resolution of [stack overflow issue](https://github.com/libp2p/js-libp2p-switch/issues/287)
+    - Volunteered to [start a ProtoSchool chapter in London](https://github.com/ProtoSchool/organizing/issues/3)
+- Blocked:
+    - 0.34 release now blocked on an [issue handling control signals](https://ci.ipfs.team/blue/organizations/jenkins/IPFS%2Fjs-ipfs/detail/PR-1803/3/pipeline/257) (also in master 😱)
+        - I am on holiday this week - would appreciate someone looking into this!
+- Next:
+    - n/a - on holiday
+
+@vasco-santos
+- Done:
+  - IPNS spec ready, reviewed and merged [ipfs/specs#184](https://github.com/ipfs/specs/pull/184)
+  - IPNS / DHT interop reviews by Alan were resolved [ipfs/interop#36](https://github.com/ipfs/interop/pull/36), [ipfs/interop#39](https://github.com/ipfs/interop/pull/39) and [ipfs/interop#47](https://github.com/ipfs/interop/pull/47)
+  - JS libp2p-daemon-client with basic commands and dht almost ready [libp2p/js-libp2p-daemon-client#1](https://github.com/libp2p/js-libp2p-daemon-client/pull/1)
+  - Helped with a lot of PRs and reviews for updating dependencies (in the context of the [stack overflow issue](https://github.com/libp2p/js-libp2p-switch/issues/287) and `js-ipfs` new release)
+- Blocked: n/a
+- Next:
+  - Preparation for libp2p team week (in Porto 🚀)
+  - Add discovery limits when using dht discovery [libp2p/js-libp2p-kad-dht#61](https://github.com/libp2p/js-libp2p-kad-dht/issues/61)
+  - PR with the first set of interop tests for libp2p using `js-libp2p-daemon-client` with `js-libp2p-daemon` and `go-libp2p-daemon`
+
+@jacobheun
+- Done:
+  - Resolved stack overflow issue in js-ipfs tests [libp2p-switch#287](https://github.com/libp2p/js-libp2p-switch/issues/287)
+    - Reviewed PRs needed for dep updates
+  - bitswap sync in async code PR [js-ipfs-bitswap#184](https://github.com/ipfs/js-ipfs-bitswap/pull/184)
+  - Caught up on open PRs from over the holiday
+  - Met with @alanshaw, @mcollina, @vasco-santos and @hugormdias to discuss switch/multistream/mplex. Discussed need to overhaul mplex.
+  	- May warrant finishing/revisiting the open pull-stream WIP PR\ [libp2p-mplex#76](https://github.com/libp2p/js-libp2p-mplex/pull/76)
+- Blocked:
+- Next:
+  - Prep for libp2p team week (all next week)
+  - Fix libp2p-crypto interface for [libp2p-crypto#129](https://github.com/libp2p/js-libp2p-crypto/issues/129)
+  - Finish initial daemon work PR [libp2p-daemon#1](https://github.com/libp2p/js-libp2p-daemon/pull/1)
+
+@hugomrdias
+- Done:
+  - Holidays and catching up
+  - Bubbling up bundle size PRs
```

Comment:
@hugomrdias where is the tracking issue for this?

Reply:
there isn't an issue, i have a PR open in all the repos

Reply:
Can we get a link to the sheet plz?